### PR TITLE
feat: builder in world unpublish scene

### PR DESCRIPTION
**Pull request by pbosio**
Add support for scene unpublishing by taking a random empty parcel from kernel and deploying it

### CODE REVIEW

File: kernel/packages/shared/apis/SceneStateStorageController/unpublishScene.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    result = { ok: false, error: `Unpublish failed ${error}` }
+    defaultLogger.error('Unpublish failed', error)
+  }
+
+  return result
+}
+
+async function getEmptySceneFiles(coordinates: string): Promise<SceneDeployment> {
+  const fullRootUrl =
+    `${location.protocol}//${location.host}${location.pathname}`.replace('index.html', '') + 'loader/empty-scenes/'
```
Comment (menduz) [inline kernel/packages/shared/apis/SceneStateStorageController/unpublishScene.ts]:
```suggestion
    new URL('loader/empty-scenes/', location).toString()
```
🤯

File: kernel/packages/decentraland-loader/lifecycle/controllers/scene.ts [hunk 2]
```diff
@@ -44,11 +44,7 @@ export class SceneLifeCycleController extends EventEmitter {
     const parcels = this.sceneStatus.get(sceneId)?.sceneDescription?.sceneJsonData?.scene?.parcels
     if (parcels) {
       // Invalidate parcel to scenes association
-      for (const parcel of parcels) {
-        this._positionToSceneId.delete(parcel)
-        this.futureOfPositionToSceneId.delete(parcel)
-      }
-      this.downloadManager.invalidateParcels(parcels)
+      this.invalidateParcels(parcels)
```
Comment (nchamo) [inline kernel/packages/decentraland-loader/lifecycle/controllers/scene.ts]: I really think that `this.sceneStatus.delete(sceneId)` should be part of the invalidation.

File: kernel/packages/shared/apis/SceneStateStorageController/BuilderServerAPIManager.ts [hunk 3]
```diff
@@ -1,20 +1,10 @@
 import { Authenticator } from 'dcl-crypto'
 import { ExplorerIdentity } from 'shared/session/types'
 import { uuid } from 'decentraland-ecs/src/ecs/helpers'
-import { ContentMapping } from '../../types'
-import { BuilderAsset, BuilderManifest, BuilderProject, BuilderScene } from './types'
+import { BuilderAsset, BuilderManifest, BuilderProject, BuilderScene, AssetId, Asset } from './types'
 import { getDefaultTLD } from 'config'
 import { defaultLogger } from '../../logger'
 
-export type AssetId = string
```
Comment (nchamo) [inline kernel/packages/shared/apis/SceneStateStorageController/BuilderServerAPIManager.ts:9]: Nice!

**Review by nchamo — APPROVED** [review]
Great work!

**Pull request merged**